x-pack/plugins/alerting/server/routes/rule/apis/unsnooze/transforms/transform_unsnooze_body/v1.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RewriteRequestCase } from '../../../../../lib';
+
+export const transformUnsnoozeBody: RewriteRequestCase<{ scheduleIds?: string[] }> = ({
```

Comment:
We want to be very explicit with these route types (since they're versioned), so we should avoid using the `RewriteRequestCase` types from now on and define them separately. It is more work and repetition but yea.